modules/nw-networkpolicy-about.adoc

```diff
@@ -48,10 +48,33 @@ spec:
   ingress: []
 ----
 
-* *Only Accept connections from pods within project*
+* Only allow connections from the {product-title} ingress router:
 +
-To make pods accept connections from other pods in the same project,
-but reject all other connections from pods in other projects:
+* To make a project allow only connections from the {product-title} ingress
+router, add the following `NetworkPolicy` object: 
```

Comment:
below it is ingress namespace

Reply:
@pecameron, does the ingress router live in the ingress namespace? This policy object was provided in OSDOCS-374 by @zhaozhanqi.